docs/kyma/04-13-local-reinstallation-deprecated.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Reinstall Kyma (Scripts)
+type: Installation
+---
+
+>**CAUTION**: This document describes a deprecated flow. Follow [this document](/root/kyma/#installation-install-kyma-locally) to install locally using the Kyma CLI.
```

Comment:
I don't think we can mark this document as deprecated - is there a way to get the same result in Kyma CLI? The document you're linking to doesn't provide reinstallation instructions. 
If you don't provide a way to reinstall Kyma using the CLI in this release then please revert this doc to its original state.

Reply:
I believe you should link to this one: `/root/kyma/#installation-reinstall-kyma`, as it provides updated instructions on reinstalling kyma using kyma CLI.